### DISCUSSION TITLE
Homepage Refresh

### DISCUSSION THREAD

**Discussion by luukee** [engagingnewsproject/enp-platform#354] [General]
@leoncai1999 @NWebb5 @SimonXie18 If you could do a little research on news centered websites and find modules you think would be good to implement on the CME home page. That would be great!

An example would be like the weather sidebar module on the [Google News](https://news.google.com/topstories?hl=en-US&gl=US&ceid=US:en) site (or something similar):
![Screen Shot 2021-07-08 at 9 24 06 AM](https://user-images.githubusercontent.com/30611098/124957874-67a42600-dfce-11eb-8371-797505487b17.png)

**Comment by julia-elias** [comment]
![image](https://user-images.githubusercontent.com/67982703/180027016-1e8cf631-8ccb-4cf4-8614-1a067d6894c8.png)

**Comment by luukee** [comment]
@julia-elias These are all great!

- More Research
  - The intro text below is a good way to preview a bit of the article.
  - The vertical layout allows for larger images, which is always a good thing.
- Quiz Creator
  - A background image with text overlay is always a good design change up. Defines that this is a different section/tool of the page.
  - Center all content with icons: 👍 
- Latest Tweets
  - I believe this is something we can set up in the plugin. Although it might require us to purchase the pro version.
- New sections
  - Data section: 👍  @katalinadeaven do we have data to input here?
  - We could also have the numbers count up on scroll-over :)
  - Most popular section 👍
